ods_ci/tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects.robot

```diff
@@ -121,8 +118,8 @@ Verify User Can Create And Start A Workbench With Existent PV Storage
     [Documentation]    Verifies users can create a workbench and connect an existent PersistenVolume
     ${pv_name}=    Set Variable    ${PV_BASENAME}-existent
     Open Data Science Project Details Page       project_title=${PRJ_TITLE}
-    Create PersistentVolume Storage    name=${pv_name}    description=${PV_DESCRIPTION}
-    ...                               size=${PV_SIZE}    connected_workbench=${NONE}    project_title=${PRJ_TITLE}
+    Create PersistentVolume Storage    name=${pv_name}    description=${PV_DESCRIPTION}    project_title=${PRJ_TITLE}
+    ...                               size=${PV_SIZE}    connected_workbench=${NONE}   existing_storage=${TRUE}
```

Comment:
why would we have a PVC with the same name already created in the project? 
I think we should keep the test the most predictable we can.

Reply:
Yes it caused issues, since the projects could have existed before.
I also added a cleanup step on Test Suite Setup, for an additional safety.
However, `existing_storage` is an optional test feature, which allows us:
1) To test UI in more ways 
2) Without facing Automation bugs if test logic did not cleanup previous PVC
3) While not not introducing any risk of "hidden bugs"